src/main/java/org/jenkinsci/plugins/compressbuildlog/BuildLogCompressor.java

```diff
@@ -28,7 +34,7 @@ public BuildLogCompressor() {
 
         @Override
         public String getDisplayName() {
-            return "Compress Build Log";
+            return "Compress Build Logs";
```

Comment:
I have no idea whether this is shown anywhere, but if it is, it should be localized while you're at it…